include/blosc2.h

```diff
@@ -1909,6 +1909,14 @@ BLOSC_EXPORT int blosc2_vlmeta_get(blosc2_schunk *schunk, const char *name,
  */
 BLOSC_EXPORT int blosc2_vlmeta_delete(blosc2_schunk *schunk, const char *name);
 
+/**
+ * @brief Get a list of all the variable-length metalayer names
```

Comment:
Missing end dot.

Reply:
Fixed in 52736067b453102049a645aa7137388b5a23e4be.